app/src/main/java/jkartkowka/jkartkwkamobile/StudentPopQuizActivity.java

```diff
@@ -41,8 +46,19 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
                 adapter.notifyDataSetChanged();
             }
         });
-        wireframe = new MultipleAnswerPopQuizWireframe(this);
 
+        interactor.getPopQuiz(new StandardGenericResponseHandler<PopQuiz>() {
+            @Override
+            public void onSuccess(PopQuiz responseObject) {
+                interactor.initialize(responseObject);
```

Comment:
Po co przechodzić z responseObjectem przez onSuccess, żeby potem to ponownie wrzucić do interactora?